try

```diff
@@ -0,0 +1,6 @@
+type int = bit<10>;
```

Comment:
This file looks like a spurious additon.

Reply:
Whoops! Deleted.